integration/bundle/deploy_test.go

```diff
@@ -226,6 +226,7 @@ func TestDeployBasicBundleLogs(t *testing.T) {
 
 	stdout, stderr := blackBoxRun(t, ctx, root, "bundle", "deploy")
 	assert.Equal(t, strings.Join([]string{
+		fmt.Sprintf("Error: failed to delete /Workspace/Users/%s/.bundle/%s/artifacts/.internal: unable to delete filer root", currentUser.UserName, uniqueId),
```

Comment:
This needs to be silenced as well.

@andrewnester We opportunistically remove the artifact path to make sure the destination is empty. We can either revisit that and keep old stuff around (IIRC, this was specifically requested by a user once), or we check for this error and skip logging if the target path doesn't exist.

Reply:
Context: https://github.com/databricks/cli/pull/442/files#diff-4091f4efa08c26352770ed7d6a05db66b5917349f22b96dc746e676d9491def2R176

It sounds a bit dangerous to call Delete on root and then have Delete function refuse the deletion (as a last resort safety measure). Would be better to avoid calling Delete in the first place.

Reply:
> It sounds a bit dangerous to call Delete on root

It's not on root but on `<root>/.internal` which is a special folder we create so we can safely remove it. As to keeping stuff around vs. cleaning up let's have a discussion as both have pros and cons

Reply:
In that case the warning message is wrong? Could you or @pietern send the code correcting the warning and downgrade it to appropriate level? It's hard for me to understand what was the intent here.